doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -229,6 +229,18 @@ Additionally, in case of the secure storage backend, the following Kconfig optio
 * :ref:`CONFIG_NCS_SAMPLE_MATTER_SECURE_STORAGE_MAX_ENTRY_NUMBER<CONFIG_NCS_SAMPLE_MATTER_SECURE_STORAGE_MAX_ENTRY_NUMBER>` - Defines the maximum number or assets that can be stored in the secure storage.
 * :kconfig:option:`CONFIG_TRUSTED_STORAGE_BACKEND_AEAD_MAX_DATA_SIZE` - Defines the maximum length of the secret that is stored.
 
+.. _ug_matter_persistent_storage_factory_reset:
+
+Factory reset
+-------------
+
+The factory reset functionality allows you to reset the device to its factory settings and remove all non-volatile data related to Matter and network.
+If you need to remove application-specific data, you must do it manually or use a dedicated callback function.
+The default implementation of the factory reset callback is located in the :file:`ncs/nrf/samples/matter/common/src/app/matter_init.cpp` file and it disables the :ref:`mpsl` to speed up flash operations during the factory reset.
```

Comment:
Maybe it's worth to mention it is done only for the single core solution, as for multi-cores it is not needed.

Reply:
@peknis I've added such a line there.